apps/extension/package.json

```diff
@@ -32,6 +32,7 @@
     "@penumbra-zone/transport-chrome": "8.0.1",
     "@penumbra-zone/transport-dom": "7.5.0",
     "@penumbra-zone/types": "26.1.0",
+    "@penumbra-zone/ui": "13.3.1",
```

Comment:
update version once new [ui package](https://github.com/prax-wallet/prax/pull/241) is published to access `showValue` prop